x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate.tsx

```diff
@@ -0,0 +1,620 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Fragment } from "react";
+import { i18n } from "@kbn/i18n";
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormLabel,
+  EuiFormRow,
+  EuiIcon,
+  EuiLink,
+  EuiPanel,
+  EuiSelect,
+  EuiSpacer,
+  EuiSuperSelect,
+  EuiText,
+  EuiTextArea,
+} from '@elastic/eui';
+import { useActions, useValues } from "kea";
+import { FormattedMessage } from '@kbn/i18n-react';
+import { docLinks } from '../../../shared/doc_links';
+
+import { AppSearchGateLogic } from "./app_search_gate_logic";
+
+const featuresList = {
+  webCrawler: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.webCrawler.featureButtonLabel',
+      {
+        defaultMessage: 'Try Open Crawler',
+      }
+    ),
+    actionLink: 'https://github.com/elastic/crawler?tab=readme-ov-file#setup',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.webCrawler.featureDescription',
+      {
+        defaultMessage:
+          "Ingest web content into Elasticsearch using a web crawler",
+      }
+    ),
+    id: 'webCrawler',
+    learnMore: 'https://github.com/elastic/crawler#readme ',
+    panelText: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.webCrawler.panelText', {
+      defaultMessage:
+        'Did you know the new self-managed Elastic open crawler is now available? You can keep your web content in sync with your search-optimized indices!',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.webCrawler.featureName', {
+      defaultMessage: 'Web Crawler',
```

Comment:
Do we want this casing, or should it be `Web crawler`?

Reply:
I don't believe so (for this and the others...) -- the design doc has them in uppercase, as well as the ones for the Workplace Search gate are upper-cased as well.

Reply:
I agree with @kderusso about the casing, I don't think we really focused on this in the design doc but sentence case makes more sense. I'm not sure it's going to make a huge difference, but if have time to fix would be nice. :)